storage/backup/backup-cli/src/bin/db-backup.rs

```diff
@@ -23,104 +23,110 @@ use backup_cli::{
     },
 };
 
-#[derive(StructOpt)]
-#[structopt(about = "Ledger backup tool.")]
+#[derive(Parser)]
+#[clap(about = "Ledger backup tool.")]
 enum Command {
-    #[structopt(about = "Manually run one shot commands.")]
+    #[clap(subcommand, about = "Manually run one shot commands.")]
     OneShot(OneShotCommand),
-    #[structopt(about = "Long running process backing up the chain continuously.")]
+    #[clap(
+        subcommand,
+        about = "Long running process backing up the chain continuously."
+    )]
     Coordinator(CoordinatorCommand),
 }
 
-#[derive(StructOpt)]
+#[derive(Parser)]
 enum OneShotCommand {
-    #[structopt(about = "Query the backup service builtin in the local node.")]
+    #[clap(
+        subcommand,
+        about = "Query the backup service builtin in the local node."
+    )]
     Query(OneShotQueryType),
-    #[structopt(about = "Do a one shot backup.")]
+    #[clap(about = "Do a one shot backup.")]
```

Comment:
This is the kind of thing where I mean the description could be better.  Using the doc comments allow for a short comment then more info e.g.

```
/// Run a single backup
///
/// This will make a backup that will load the entire database
/// at one time, and then state sync would catch up the rest.
```